cora/signal/lss.py

```diff
@@ -875,6 +1003,335 @@ def za_density_grid(
     return out
 
 
+class FingersOfGod(task.SingleTask):
+    r"""Apply a smoothing to approximate the effects of Fingers of God.
+
+    This applies an exponential smoothing kernel which is equivalent to a *squared
+    Lorentzian* suppression in power spectrum space which is a reasonably common model.
+
+    Attributes
+    ----------
+    alpha_FoG : float
+        A parameter to control the strength of the effect by adjusting the smoothing
+        scale. A value of 1 (default) applies the nominal smoothing, 0 turns the
+        smoothing off entirely, and any other values adjust the scale appropriately.
+    model : {'HI', 'LRG', 'ELG', 'QSO'}
+        Use an inbuilt model for the Fingers of God smoothing. If `None` (default), a
+        specific model is expected to be set via `FoG_coeff` and `z_eff`.
+    FoG_coeff : list
+        A list of coefficient in a polynomial of `FoG_coeff[i] * (z - z_eff)**i`. If
+        `None` default, a `model` must be set.
+    z_eff : float
+        The effective redshift of the polynomial expansion.
+
+    Notes
+    -----
+    There are builtin models for three tracers: Quasars, LRGs, and ELGs. For these a
+    measurement from actual data has been used to provide an overall normalisation and a
+    satellite-HOD-weighted average over `sigma_v(M)` has been used to provide a redshift
+    dependence (the output of this calculation is renormalised by the measurement).
+
+    For LRGs we use a measurement from Gil-Marin et al. 2021 [1]_. The baseline model
+    uses a HOD from Alam et al. 2020 [8]_, and the alt model from Zhai et al. 2017 [2]_.
+
+    For ELGs we use a measurement from de Mattia et al. 2021 [3]_. The baseline model
+    uses the HMQ HOD from Alam et al. 2020 [8]_ and alt model the `HOD-3` HOD from Avila
+    et al. 2020 [4]_.
+
+    For QSOs we use a measurement from Neveux et al. 2021 [5]_. However, this combines
```

Comment:
Our QSO model has actually changed to the Zarrouk one. Here's a suggested description:
```
For QSOs we use a measurement from Zarrouk et al. 2018 [5]_ that attempts to separate the
physical FoG damping from the damping caused by redshift errors. Their damping is modelled
by a Gaussian, and we have converted their stated damping scale into one that is roughly
equivalent for a double Lorentzian. We have taken the average of the values from their
3-multipole and 3-wedge HOD values. The baseline HOD is also from Alam et al. [8]_, and the
alt model uses the `4KDE+15eBOSS` QSO HOD from Eftekharzadeh et al. 2020 [6]_.
```